calculate_baseline_metrics.py

```diff
@@ -122,31 +122,42 @@ def parse_args():
     parser = argparse.ArgumentParser(
         description='AtacWorks script to calculate metrics on batched data.')
     parser.add_argument('--label_file', type=str,
-                        help='Path to hdf5/bw file containing labels')
+                        help='Path to hdf5/bigWig file containing labels',
+                        required=True)
     parser.add_argument('--test_file', type=str,
-                        help='Path to hdf5/bw file containing labels. Assumed \
-                                to be present in label_file by default.')
-    parser.add_argument('--peak_file', type=str,
-                        help='Path to hdf5/bw file with peak labels. Assumed \
+                        help='Path to hdf5/bigWig file containing labels. Assumed \
```

Comment:
the description of label_file and test_file look the same right ? Why do we need a test_file if we assume labels are in label_file by default ?

Reply:
The label file can be an h5 file generated by `bw2h5.py`. If it is generated with `--nolabel`, it will not have labels. In that case, we need the labels to be supplied as a separate file (test_file). Rephrased this to make it clearer.

Reply:
Fixed.